Tests/test_font_crash.py

```diff
@@ -17,8 +15,7 @@ def _fuzz_font(self, font: ImageFont.FreeTypeFont) -> None:
             draw.multiline_textbbox((10, 10), "ABC\nAaaa", font, stroke_width=2)
             draw.text((10, 10), "Test Text", font=font, fill="#000")
 
-    @skip_unless_feature("freetype2")
+    @skip_unless_feature_version("freetype2", "2.12.0")
```

Comment:
Why change the version here?

Reply:
The test intended to catch `OSError: Bitmap missing for glyph` from https://github.com/python-pillow/Pillow/blob/b6f90c47325f4d6cb507269b51cf309319eca8d7/src/_imagingft.c#L1028-L1032

For FreeType2 versions before 2.12.0, `FT_New_Face` is returning an `Unknown_File_Format` error. It does that even on main at the moment, it's just not obvious because both are `OSError`. If I [increase the specificity of the test](https://github.com/radarhere/Pillow/commit/8663c46d7d1aedd199130068539df5575f36d414), [you can see this.](https://github.com/radarhere/Pillow/actions/runs/10536876212)

FreeType2 2.12.0 was released in March 2022, before #6846, so I expect this has been the case the whole time. Now that I've fixed the `OSError: Bitmap missing for glyph` error, the other error becomes obvious.

This was a font generated by OSS-Fuzz, so its purpose is not to be valid, but merely to trigger a security problem. We could modify the file to be valid, but in principle it would seem to be better to not modify test scenarios over time.

I suppose I could modify the test to run on FreeType2 < 2.12.0 as well, and just catch the error for that scenario if you want.

Reply:
Thank you for the explanation.